### PR TITLE
Update LicenseCheckBearTest.py

### LINKED ISSUE #1704: Use with to load the file
In https://github.com/coala/coala-bears/blob/master/tests/general/LicenseCheckBearTest.py#L18,
file is opened without using `with` statement. It doesn't ensure the closing of file.

### PR DESCRIPTION
**Pull request by captaincrunch21**
Fixes #1704

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: tests/general/LicenseCheckBearTest.py [hunk 1]
```diff
@@ -15,7 +15,8 @@ def get_testfile_path(name):
 
 
 def load_testfile(name):
-    return open(get_testfile_path(name)).readlines()
+    with open(get_testfile_path(name)).readlines() as lines:
```
Comment (shreyans800755) [inline tests/general/LicenseCheckBearTest.py]: This is not correct syntax.
Please refer https://github.com/coala/coala-bears/pull/1691/commits/5246a7f819cf98861a66f9befa50f21d999ff7dc#diff-01d6f9cb6e2d53a776a4b7ec3388b233R19 and correct it.
This is causing breaks in CI builds.

**Comment by shreyans800755** [comment]
Update commit message as per guidelines given in http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

**Comment by jayvdb** [comment]
Note we have two PRs for the same issue https://github.com/coala/coala-bears/issues/1704

**Comment by jayvdb** [comment]
Hi @captaincrunch21 , another newcomer has been assigned to fix https://github.com/coala/coala-bears/issues/1704 .
Please follow the newcomer guide , first step is joining the coala organisation.  Then find another newcomer issue to work on, and ask to be assigned to the issue.

**Pull request closed without merging**